app/components/meeting/edit-meeting-modal.hbs

```diff
@@ -115,16 +115,18 @@
           </div>
         </AuFormRow>
       {{/if}}
-      <AuFormRow>
-        <Auk::Label>
-          {{t "secretary-for-meeting"}}
-        </Auk::Label>
-        <Mandatees::MandateeSelector
-          @onSelectMandatee={{fn (mut this.secretary)}}
-          @selectedMandatees={{this.secretary}}
-          @visibleRoles={{this.visibleRoles}}
-        />
-      </AuFormRow>
+      {{#if this.enableDigitalAgenda}}
+        <AuFormRow>
+          <Auk::Label>
+            {{t "secretary-for-meeting"}}
+          </Auk::Label>
+          <Mandatees::MandateeSelector
+            @onSelectMandatee={{fn (mut this.secretary)}}
+            @selectedMandatees={{this.secretary}}
+            @visibleRoles={{this.visibleRoles}}
+          />
+        </AuFormRow>
+      {{/if}}
```

Comment:
This hides the component, but the `js` file had numerous references to the secretary as well. Adding the flag there as well for consistency